src/client/components/LandingPage.tsx

```diff
@@ -32,6 +33,9 @@ export const LandingPage: React.FC = () => {
               <br /> for Developers
             </h1>
             <p className="subtitle">A free, open-source notes app for the web.</p>
+            {isMobile && (
+              <p className="p-mobile">TakeNote is not currently supported for mobile devices.</p>
+            )}
             <div className="new-signup">
               {isDemo ? (
```

Comment:
The demo button will still display if someone is on mobile. Is that the intent? I was under the assumption that it would be hidden if a user was on mobile.

Reply:
@kodyclemens i've no idea. Maybe, @taniarascia could confirm it?

Reply:
The button should not be displayed.

Reply:
Done.